src/App.tsx

```diff
@@ -1,26 +1,209 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useState, useEffect } from 'react';
+import cn from 'classnames';
 
-const USER_ID = 0;
+import { UserWarning } from './UserWarning';
+import { USER_ID } from './api/todos';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import TodoList from './components/TodoList';
+import { Status } from './types/Status';
+import * as postService from './api/todos';
+import { Todo } from './types/Todo';
+import { handleError } from './components/Error';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<Status>(Status.All);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [inputDisabled, setInputDisabled] = useState(false);
+  const [loading, setLoading] = useState(false);
```

Comment:
- Boolean variables should start from `is` or `has`.

Hint: instead you can create a state with an array of loading todo ids. So you can add and remove ids of todos that currently loading and show loaders for them. It can help with the next task